docs/adrs/0549-composite-run-steps.md

```diff
@@ -16,11 +16,32 @@ We don't want the workflow author to need to know how the internal workings of t
 
 A composite action is treated as **one** individual job step (this is known as encapsulation).
 
-
 ## Decision
 
 **In this ADR, we only support running multiple run steps in an Action.** In doing so, we build in support for mapping and flowing the inputs, outputs, and env variables (ex: All nested steps should have access to its parents' input variables and nested steps can overwrite the input variables).
 
+## Composite Run Steps Features
```

Comment:
should this be three `###`

only `Context`, `Decision`, and `Consequences` are `##`

Reply:
good point. Thanks!